src/ui/src/components/core/content/CoreChatBot/CoreChatbotMessage.vue

```diff
@@ -34,6 +82,19 @@
 			</div>
 		</div>
 	</div>
+
+	<!-- Image Modal -->
+	<WdsModal
+		v-if="showImageModal"
+		:display-close-button="true"
+		@close="closeImageModal"
+	>
+		<img
+			:src="modalImageUrl"
+			:alt="'Full size image'"
+			class="image-modal-img"
+		/>
+	</WdsModal>
```

Comment:
You can simply use `modalImageUrl`, and display the image if this ref is filled

```suggestion
	<WdsModal
		v-if="modalImageUrl"
		:display-close-button="true"
		@close="modalImageUrl = undefined"
	>
		<img
			:src="modalImageUrl"
			:alt="'Full size image'"
			class="image-modal-img"
		/>
	</WdsModal>
```